### ISSUE TITLE
jdbc40 logs some messages without resolving bundles and spams server.log

### ISSUE THREAD

**Issue by dmatej** [payara/Payara#714]
When using ifxjdbc without 30 wrapper, log is spammed by 
`jdbc.unable_to_set_client_info` and `jdbc.unable_to_get_client_info` messages.

```
            if (isSupportClientInfo()) {
                defaultClientInfo = getClientInfo();
            }
        } catch (Throwable e) {
            _logger.log(Level.INFO, "jdbc.unable_to_get_client_info", e.getMessage());
            if(_logger.isLoggable(Level.FINEST)) {
                _logger.log(Level.FINEST, "jdbc.unable_to_get_client_info", e);
            }
        }

```
1) Avoid catching Throwable
2) If driver does not support clientInfo, there is no problem, there is nothing to cache (javadoc:  cache the default client info which can will set back during close() as this connection may be re-used by connection pool of application server).

I can fix it ...

**Comment by dmatej** [comment]
Aha ... the problem is that the jdbc-core's LogStrings.properties are not found because classloader finds com.sun.logging.enterprise.resource.resourceadapter.LogStrings.properties sooner.
This system is a big mess ... logging dependency hell.
I think the best solution would be to allow avoiding the getResourceBundleNameForDomainRoot magic and to prefer more straightforward access to resource bundles.
**What do you think?**

The final solution should be to reimplement the whole logging system, but this would take much more time ... many bundle keys are in other place than expected, many are unused, many are loaded from dependencies ... and many messages are not in bundles.

I can fix this and then I will fix related problem I have found.

**Comment by dmatej** [comment]
Ha, found a better solution - I have moved the LogStrings.properties in connectors-runtime so it is not in conflicting path and it is still found by the LogDomains.

**Comment by smillidge** [comment]
I think we need a task to redo all the resource bundles :-(

**Comment by dmatej** [comment]
This issue should be fixed here: https://github.com/payara/Payara/pull/716
I have tested it manually, it seems it logs what is expected :-)

**Comment by mikecroft** [comment]
Fixed in #716
